src/js/vic-v2/components/DD214Description.jsx

```diff
@@ -7,7 +7,7 @@ export default function DD214Description() {
       <p>If you have more than one discharge document, please upload the one with the highest character of discharge.</p>
       <p>If you don’t have a digital copy of your discharge document, you can scan or take a photo of it and then upload it to your computer or phone. The digital file could be a .pdf file or other photo file format, like a .jpeg or .png.</p>
 
-      <div>File types you can upload: .pdf, .jpeg, or .png</div>
+      <div>File types you can upload: .pdf, .jpeg, .gif, .tif, or .png</div>
```

Comment:
Should we use .tiff here since we use .jpeg or the three letter extensions for each?